inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/pooling_gpu_b_fs_zyx_fsv16_imad.cl

```diff
@@ -44,6 +55,243 @@ inline ACCUMULATOR_TYPE FUNC(apply_pooling)(ACCUMULATOR_TYPE tmp, ACCUMULATOR_TY
 #endif
 }
 
+#if GLOBAL_POOLING
+__attribute__((intel_reqd_sub_group_size(FEATURE_SLICE_SIZE)))
+__attribute__((reqd_work_group_size(1, LWS, 1)))
```

Comment:
Looks like this requirements may cause error since LWS is not aligned to FEATURE_SLICE_SIZE